src/classes/configuration_box.cpp

```diff
@@ -125,18 +130,20 @@ void Configuration::applySizeFactor(const ProcessPool &procPool, const Potential
          *  -- Otherwise, check energy - if it is negative, reduce requested size factor
          *  -- If energy is positive, don't change anything
          */
-        if (fabs(requestedSizeFactor_ - 1.0) < 1.0e-5)
+        if (fabs(requestedSizeFactor_.value_or(defaultSizeFactor_) - 1.0) < 1.0e-5)
             break;
         else if (EnergyModule::interMolecularEnergy(procPool, this, potentialMap) <= 0.0)
         {
-            requestedSizeFactor_ *= reductionFactor;
-            if (requestedSizeFactor_ < 1.0)
+            requestedSizeFactor_ = requestedSizeFactor_.value_or(defaultSizeFactor_) * reductionFactor;
+            if (requestedSizeFactor_.value_or(defaultSizeFactor_) < 1.0)
                 requestedSizeFactor_ = 1.0;
-            Messenger::print("Intermolecular energy is zero or negative, so reducing SizeFactor to {}\n", requestedSizeFactor_);
+            Messenger::print("Intermolecular energy is zero or negative, so reducing SizeFactor to {}\n",
+                             requestedSizeFactor_.value_or(defaultSizeFactor_));
```

Comment:
It definitely has a value here as well as in a few lines above, so:
```suggestion
                             *requestedSizeFactor_);
```